Jenkinsfile

```diff
@@ -218,7 +218,7 @@ lock(resource: "Lisk-Core-Nodes", inversePrecedence: true) {
 			"Functional Peer - Blocks" : {
 				node('node-02'){
 					sh '''
-					export TEST=test/api/peer.blocks.js TEST_TYPE='FUNC'
+					export TEST=test/api/peer/peer.blocks.ws.js TEST_TYPE='FUNC'
```

Comment:
after discussion with @karmacoma  we decided to move WS tests to `transport` folder and rename them accordingly